src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusSerializer.cs

```diff
@@ -396,6 +397,40 @@ public static int WriteTags(byte[] buffer, int cursor, Metric metric, ReadOnlyTa
         return cursor;
     }
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static int WriteTargetInfo(byte[] buffer, int cursor, Resource resource)
```

Comment:
Given resource information is static throughout, a nice perf optimization can be achieved by caching the serialized version of that and re-using it as-is, in the right place, for each scrape! Not a blocker for the PR.

Reply:
Yep, i'm working on adding this in at the moment.